webservices/templates/swagger-ui.html

```diff
@@ -30,14 +29,85 @@
 </head>
 
 <body>
-    <div id="swagger-ui"></div>
+
+<div id="swagger-ui"></div>
+ <script type="text/javascript">
+   /* * * CONFIGURATION VARIABLES: EDIT BEFORE PASTING INTO YOUR WEBPAGE * * */
+   var apiUmbrellaSignupOptions = {
+     // Pick a short, unique name to identify your site, like 'gsa-auctions'
+     // in this example.
+     registrationSource: 'fec-api',
+
+     // Enter the API key you signed up for and specially configured for this
+     // API key signup embed form.
+     apiKey: "{{ api_key_signup_key }}",
+     // document.write(apiKey.toLocaleString( ));
+     // Provide a URL or e-mail address to be used for customer support.
+     //
+     // The format for e-mail addresses can be given as either
+     // 'example@example.com' or 'mailto:example@example.com'.
+     contactUrl: 'APIinfo@fec.gov',
+
+     // Provide the name of your developer site. This will appear in the
+     // subject of the welcome e-mail as "Your {{siteName}} API key".
+     siteName: 'openFEC',
+
+     // Provide a sender name for who the welcome email appears from. The
+     // actual address will be "noreply@api.data.gov", but this will
+     // change the name of the displayed sender in this fashion:
+     // "{{emailFromName}} <noreply@api.data.gov>".
+     emailFromName: 'openFEC',
+
+     // Provide an example URL you want to show to users after they signup.
+     // This can be any API endpoint on your server, and you can use the
+     // special {{api_key_signup_key}} variable to automatically substitute in the API
+     // key the user just signed up for.
+     // exampleApiUrl: 'https://api.open.fec.gov/v1/legal/search/?api_key_signup_key={{api_key_signup_key}}',
```

Comment:
It looks like the example API call that goes out is still NREL - can you uncomment this line please?

Reply:
There's an issue with the variable passing the api umbrella  key rather than inserting the NEW key. We will work it out in the next iteration!

Reply:
👍 From this page: https://api.data.gov/docs/agency-manual/ it looks like we need to use `{{api_key}}`

```
     // Provide an example URL you want to show to users after they signup.
     // This can be any API endpoint on your server, and you can use the
     // special {{api_key}} variable to automatically substitute in the API
     // key the user just signed up for.
     exampleApiUrl: 'https://api.data.gov/gsa/auctions?api_key={{api_key}}&format=JSON'
```

Reply:
Hm, it looks like that's just a different version of above, sorry if that doesn't work.

Reply:
Correct syntax to resolve for the FEC's architecture:

```
exampleApiUrl: 'https://api.open.fec.gov/v1/legal/search/?api_key={{ '{{api_key}}' }}'
```

(need to escape the already-used `{{api_key}}` variable)